### PR TITLE
Flask/couch db

### PR DESCRIPTION
**Pull request by anantmittal**
@jayfo 

`Pipfile` - added autopep8 package
`invoke/database_tests.py` - added a helper task function which prints out which user doesn't exist in test accounts yaml file.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -11,20 +11,10 @@ name = "pypi"
 [packages]
 pipenv = "*"
 requests = "*"
-
```
Comment (jayfo) [inline Pipfile:14]: I'm guessing pipenv automatically did this when you installed autopep8?

Let's not do this destruction of my documentation?

File: Pipfile [hunk 2]
```diff
@@ -11,20 +11,10 @@ name = "pypi"
...
-[packages.aws-infrastructure]
```
Comment (jayfo) [inline Pipfile:15]: I'm guessing pipenv automatically did this when you installed autopep8?

Let's not do this destruction of my documentation?

Comment (jayfo) [inline Pipfile:15]: I'm guessing pipenv automatically did this when you installed autopep8?

Let's not do this destruction of my documentation?

File: server_flask/database.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    def load(baseurl: str, admin_user: str, admin_password: str):
+
+        return CouchDBClientConfig(
+            baseurl=baseurl,
+            admin_user=admin_user,
+            admin_password=admin_password,
+        )
+
+    @staticmethod
+    def initialize(self):
```
Comment (jayfo) [inline server_flask/database.py:31]: The database initialize only ever happens one time, when I set up the cluster / database.  The invoke task seems the right place for that, don't think we need it here in Flask at all.

Comment (anantmittal) [reply server_flask/database.py]: Removed this and pushed the changes.

File: Pipfile (hunk 2 above)
Comment (anantmittal) [reply Pipfile]: Done. Reverted to its old state.

File: tasks/database_tests.py [hunk 4]
```diff
@@ -231,7 +236,8 @@ def delete_accounts(context):
     #
     # Obtain our connection information and admin credentials
     #
-    couchdb_client_config = CouchDBClientConfig.load(couchdb_client_config_path=COUCHDB_CLIENT_CONFIG_PATH)
+    couchdb_client_config = CouchDBClientConfig.load(
```
Comment (jayfo) [inline tasks/database_tests.py:239]: I've never used autopep8, and I kind of hate this code format (i.e., the way it handles parentheses).

But I'm not opposed to code formatting, might even think it's a good idea.

What do you think about Black?

https://black.readthedocs.io/en/stable/getting_started.html#do-you-like-the-black-code-style

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html

**Pull request merged**